### PR TITLE
publish snapshots for 2.7.x branch

### PR DESCRIPTION
**Pull request by renatocaval**
This set interplay to default to 2.7.x and make it possible to override it as well. 

Resolves https://github.com/playframework/playframework/issues/8946

### CODE REVIEW

File: bin/snapshots [hunk 1]
```diff
@@ -23,7 +23,7 @@ declare -a args
 
 while [[ $# -gt 0 ]] ; do
   case "$1" in
-    -b|--branch) branch=$2; shift 2 ;;
+    -b|--branch) BRANCH=$2; shift 2 ;;
```
Comment (renatocaval) [inline bin/snapshots]: Seems that this was broken. We read arg `--branch` and assign its value to `branch` variable, but this is never read. 

I guess the intention was to override `BRANCH`

**Comment by mkurz** [comment]
:+1: Will this also enable snapshots from the master (2.8) branch again?

**Comment by renatocaval** [comment]
> Will this also enable snapshots from the master (2.8) branch again?

That's a good question. If we assume that the reason why we stopped to produce snapshots for 2.7.x is because we created the 2.7.x branch, then I would expected that it would have automatically started to build snapshots for 2.8.0 (`master`). 

But that seems not to be the case. I maybe be overlooking something.

**Comment by marcospereira** [comment]
> 👍 Will this also enable snapshots from the master (2.8) branch again?

Yes, we think so. Problem is with this line:

https://github.com/playframework/interplay/blob/54ba5be655dbb61f41355520dc3304a4657deec4/bin/snapshots#L86

Because it is failing to download a dependency for 2.10 (scala-parse-combinators) and then the whole script is failing.

File: bin/snapshots [hunk 2]
```diff
@@ -83,7 +83,7 @@ echo --- twirl $TWIRL_VERSION
 echo
 
 cd $DEPLOY/twirl
-build $TWIRL_VERSION ++2.11.11 ++2.12.2 publish ++2.10.6 publish plugin/publish
```
Comment (renatocaval) [inline bin/snapshots]: twirl is failing when we try to build with 2.10.6 and as a result none of the snapshots are being published. 
https://github.com/playframework/twirl/pull/190/files#diff-fdc3abdfd754eeb24090dbd90aeec2ceR104

**Comment by marcospereira** [comment]
@mkurz https://github.com/playframework/interplay/pull/57/commits/048bc061ce7cb18b407469541351839d6dbeb992 should fix the problem.

**Pull request merged**